src/cta-banner/block.spec.js

```diff
@@ -0,0 +1,144 @@
+jest.setTimeout(30000)
+import {
+    createNewPost,
+    enablePageDialogAccept,
+    getEditedPostContent,
+    insertBlock,
+} from '@wordpress/e2e-test-utils';
+
+import {
+    clickElementByText,
+    blockStartup,
+    updateRangeInput,
+    clickCheckbox,
+    clickRadio,
+    openSidebarPanelWithTitle
+} from '../test-helpers'
+
+const block = {blockTitle: 'CTA Banner', blockName: 'purdue-blocks/cta-banner'}
+
+describe( '🔬 Proofpoint Block', () => {
```

Comment:
The block name in the description is not correct. Other than that it looks good!